.claude/docs/domains.md

```
@@ -0,0 +1,114 @@
+# 도메인 모델
+
+## 핵심 엔티티 관계
+
+```
+Host (작가)
+  └── Space (전시 공간) 1:1
+        ├── Product (작품) 1:N
+        │     └── ProductPhoto 1:N
+        └── GuestBookCard (방명록) 1:N
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)